schemas/compat-data-schema.md

```diff
@@ -120,7 +120,7 @@ The currently accepted browser identifiers should be declared in alphabetical or
 
 - `chrome`, Google Chrome (on desktops)
 - `chrome_android`, Google Chrome (on Android)
-- `edge`, MS Edge (on Windows), based on the EdgeHTML version
+- `edge`, MS Edge (on Windows), based on the EdgeHTML version, and later on Chromium versions
```

Comment:
suggestions: un-abbreviate Microsoft, use singular "version" in both cases, and describe the demarcation between EdgeHTML and Chromium

```suggestion
- `edge`, Microsoft Edge (on Windows), based on the EdgeHTML version (before version 79), and later on the Chromium version
```